<?xml version="1.0"?>
<launch>

  <arg name="mode" default="$(env MOVEL_MODE)" doc="mode: [deployment, development]"/>

  <node pkg="task_supervisor" type="launch_manager_node.py" name="launch_manager" output="screen" required="true">
    <rosparam command="load" file="$(find task_supervisor)/config/launch_manager.yaml"/>
  </node>

</launch>
